src/app/asset-page/artstor-viewer/artstor-viewer.component.scss

```diff
@@ -33,23 +33,14 @@ $desktop: 992px;*/
     overflow: hidden;
 }
 
-.asset-viewer:hover .asset-viewer__buttons  {
```

Comment:
If you were not hovering over the viewer the zoom/in out controls etc would auto hide themselves. This is not very accessibility friendly, and it was hiding the focus. We talked with Nick and determined that we will just always show the controls from here on out.